hub/apps/whats-new/latest-doc-updates.md

```diff
@@ -18,54 +18,50 @@ For the latest Windows Developer Documentation news, or to reach out to us with
 
 Don't forget to visit the [Windows Developer Center](https://developer.microsoft.com/windows/), where we highlight some of the latest technologies, frameworks and news for Windows developers.
 
-*Many thanks to everyone who has contributed to the documentation. Your corrections and suggestions are very welcome! For information on contributing, please see our [contributor guide](/contribute/).*
+_Many thanks to everyone who has contributed to the documentation. Your corrections and suggestions are very welcome! For information on contributing, please see our [contributor guide](/contribute/)._
 
 Highlights this month include:
 
 ## Microsoft Build announcements for Windows developers
 
 * [Dev Home](/windows/dev-home/): Monitor your work in the centralized dashboard, GitHub and System performance widgets. Get setup and onboard new projects with the Machine configuration tool. Learn about Dev Home’s:  
-    * [Dashboard widgets](/windows/dev-home/), including the ability to set up customized GitHub feeds, 
-    * [Machine configuration tool](/windows/dev-home/setup) for setting up a new machine or onboard a new project, and  
-    * [Customizable extensions](/windows/dev-home/extensions), with the ability to set up GitHub notifications or build and share your own Open Source extension for Dev Home. 
+  * [Dashboard widgets](/windows/dev-home/), including the ability to set up customized GitHub feeds,
+  * [Machine configuration tool](/windows/dev-home/setup) for setting up a new machine or onboard a new project, and  
+  * [Customizable extensions](/windows/dev-home/extensions), with the ability to set up GitHub notifications or build and share your own Open Source extension for Dev Home.
 
-* [Dev Drive](/windows/dev-drive/): Improve your dev workload performance using this new storage format, using ReFS and specifically designed for developer scenarios, with the ability to designate trust, manage antivirus configurations, and attach security filters. 
-    * Microsoft Defender has a new [performance mode](/microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-antivirus-performance-mode?view=o365-worldwide) specifically designed for Dev Drive. 
+* [Dev Drive](/windows/dev-drive/): Improve your dev workload performance using this new storage format, using ReFS and specifically designed for developer scenarios, with the ability to designate trust, manage antivirus configurations, and attach security filters.
+  * Microsoft Defender has a new [performance mode](/microsoft-365/security/defender-endpoint/microsoft-defender-endpoint-antivirus-performance-mode?view=o365-worldwide) specifically designed for Dev Drive.
```

Comment:
Is this view parameter needed? See build suggestion. It should either be removed or add `&preserve-view=true` to the URL if it's needed.

Reply:
Uhm, honestly I have no idea, haha. I just let VS Code (Markdown plug-in) do a quick "Format Document" 😅

Reply:
OK. Probably fine to leave it for now since it's not something you added.